contracts/workflows/GroupingWorkflows.sol

```diff
@@ -341,4 +344,219 @@ contract GroupingWorkflows is
     /// @dev Hook to authorize the upgrade according to UUPSUpgradeable
     /// @param newImplementation The address of the new implementation
     function _authorizeUpgrade(address newImplementation) internal override restricted {}
+
+    ////////////////////////////////////////////////////////////////////////////
+    //                   DEPRECATED, WILL BE REMOVED IN V1.4                  //
+    ////////////////////////////////////////////////////////////////////////////
+
+    /// @notice Mint an NFT from a SPGNFT collection, register it with metadata as an IP, attach
+    /// license terms to the registered IP, and add it to a group IP.
+    /// @notice THIS VERSION OF THE FUNCTION IS DEPRECATED, WILL BE REMOVED IN V1.4
```

Comment:
Does it need the 2nd @notice?

Reply:
Not required,  but I added it as an extra precaution to make sure the warning is more visible in case the first notice is overlooked.